src/rsa/rsa.rs

```diff
@@ -67,11 +67,23 @@ macro_rules! rsa_pkcs1_padding {
         pub static $PADDING_ALGORITHM: padding::PKCS1 =
             padding::PKCS1 {
                 digest_alg: $digest_alg,
-                digestinfo_prefix: $digestinfo_prefix,
+                digestinfo_prefix: $digestinfo_prefix
```

Comment:
The comma is there so you don't have to change the line if you add another field. Did you have a particular reason for dropping it? Personally I think we should keep it (but then I introduced it, I think).